lib/modules/dosomething/dosomething_campaign/dosomething_campaign.theme.inc

```diff
@@ -247,7 +247,7 @@ function dosomething_campaign_preprocess_hot_module(&$vars, &$wrapper) {
   $start_date = $vars['hm_start_date'];
   $end_date = $vars['hm_end_date'];
   $readable_end_date = $vars['hm_readable_end_date'];
-  $time_left = dosomething_campaign_get_hot_time_left($node, $vars);
+  $time_left = dosomething_campaign_get_hot_time_left($vars);
```

Comment:
the function definition is `dosomething_campaign_get_hot_time_left($node, &$vars = NULL) `

and you are calling it here without a NULL `$node` value, this will treat the vars array as the node object
dosomething_campaign_get_hot_time_left($vars);